language_names/scraper-sil.py

```diff
@@ -6,6 +6,8 @@
 import sys
 
 sil_names = 'http://www-01.sil.org/iso639-3/iso-639-3.tab'
+
+user_agent = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36'}  # noqa: E501
```

Comment:
```suggestion
headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36'}  # noqa: E501
```

(also modify this below, please)

Reply:
Okay I will change this